packages/pluggableWidgets/slider-web/tsconfig.json

```diff
@@ -0,0 +1,9 @@
+{
+    "extends": "@mendix/pluggable-widgets-tools/configs/tsconfig.base",
+    "baseUrl": "./",
+    "include": ["./src", "./typings", "node_modules/@mendix/pluggable-widgets-tools/typings"],
+    "compilerOptions": {
+        "typeRoots": ["node_modules/@types", "../../../node_modules/@types"],
+        "types": ["jest"]
+    }
```

Comment:
This is the first time I see this in a widget `tsconfig.json`, why do we need this? Can't we do without?

Reply:
I think is from the old widget. We can replace with:
```
{
    "extends": "@mendix/pluggable-widgets-tools/configs/tsconfig.base",
    "baseUrl": "./",
    "include": ["./src", "./typings"]
}
```